android/app/src/main/java/chat/rocket/android/MainApplication.java

```diff
@@ -82,6 +82,20 @@ public ReactNativeHost getReactNativeHost() {
   public void onCreate() {
     super.onCreate();
     SoLoader.init(this, /* native exopackage */ false);
+    setupContext();
+  }
+
+  private void setupContext() {
```

Comment:
remove this code block

Reply:
@djorkaeffalexandre Do you still need it here?